src/test/java/Homework20.java

```diff
@@ -0,0 +1,71 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest {
+
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+        String uniquePlaylist = "1-HW19-Playlist";
+        String deletePlaylistMsg = "Deleted playlist \"1-HW19-Playlist.\"";
+
+        //Pre-condition
+        //Login
+        provideEmail("demo@class.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+
+        //Create empty playlist
+        createPlaylist(uniquePlaylist);
+
+        Thread.sleep(4000);
+
+        //Click on empty playlist created
+        clickOnPlaylist();
+
+        //delete selected playlist
+        clickOnDeleteBtn();
+
+        Thread.sleep(2000);
+
+        //Expected Results
+        WebElement notificationMsg = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.alertify-logs div.success")));//driver.findElement(By.cssSelector("div.alertify-logs div.success"));
+
+        Assert.assertEquals(deletePlaylistMsg,notificationMsg.getText());
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.